Makefile

```diff
@@ -3,7 +3,7 @@ COMPANY=giantswarm
 REGISTRY=gsoci.azurecr.io
 SHELL=bash
 MARKDOWNLINT_IMAGE=ghcr.io/igorshubovych/markdownlint-cli:v0.35.0@sha256:22cf4699a448a7bbc311a940e0600019423d7671cbedae9c35cd32b51f560350
-VALE_IMAGE=gsoci.azurecr.io/giantswarm/vale:v3.4.0@sha256:fe9c173ef6dcd782cefa53cad69d1684bc317b12188b03299b0d29aecf265a59
+VALE_IMAGE=quay.io/giantswarm/vale:v3.4.0
```

Comment:
Otherwise it cannot be running from a external person (needs to login to registry)

Reply:
Everything on `gsoci.azurecr.io` is public, no auth required. I have experienced problems with pulling in the past after I was authenticated against the registry. Not sure what caused it, but `az acr login ...`solved it then.

Reply:
let me check